tests/Aspire.Hosting.Redis.Tests/RedisFunctionalTests.cs

```diff
@@ -166,7 +158,7 @@ public async Task VerifyWithRedisInsight()
 
         await pipeline.ExecuteAsync(async (ct) =>
```

Comment:
You should be able to remove this now right?

Reply:
I think so.

Reply:
For removing this i think we should use waitfor for Redis Insight

Reply:
@davidfowl Redis Insight doesn't log anything useful for using `WaitForText` so I just used `ResourceNotificationService` to ensure resource starting.